testing/prow/benchmarking.sh

```diff
@@ -0,0 +1,78 @@
+#! /usr/bin/env bash
+
+set -x
+set -o pipefail
+set -o errexit
+set -o nounset
+
+EPOCHS=3
+THRESHOLD=0.05
+MINSR=150.0
+MAXDR=30
+
+assess_benchmark_stats () {
+    artifact_dir=${ARTIFACT_DIR:-"/tmp/ci-artifacts_$(date +%Y%m%d)"}
+    sample_rate=$(cat "${artifact_dir}/benchmarking_run_ssd_sample_rate.log")
+    bench_duration=$(cat "${artifact_dir}/benchmarking_run_ssd_bench_duration.log")
+    if [[ $MINSR -gt $sample_rate ]] ; then
+        echo "Sample rate below minimum expected"
+        return 0
+    else
+        echo "Sample rate test passed!"
+    fi
```

Comment:
you can remove the `else`, that's a "guard" pattern:

```
if not test:
  return err
...
```

which avoid multiple levels of indentation (in the worst case):
```
if test:
   ... 
   do 
   many 
   things

return 0
```
or
```
if test:
  do something
    if test2:
       do something2

return
```
this one can be rewritten with all the important code at level 0